<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Birthtotal</class>
 <widget class="QWidget" name="Birthtotal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>统计生日月份</string>
  </property>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>统计</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>131</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>      一月：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>      二月：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>      三月：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>      四月：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>      五月：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>      六月：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>      七月：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>      八月：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>      九月：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>      十月：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>    十一月：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>    十二月：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>258</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTextBrowser" name="t_1">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="t_2">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="t_3">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="t_4">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="t_5">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="t_6">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="t_7">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="t_8">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="t_9">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="t_10">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="t_11">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="t_12">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
